contracts/Genesis.sol

```diff
@@ -196,31 +205,25 @@ contract Genesis is ERC721, VRFConsumerBase, Ownable {
     }
 
     /**
-     * Enumerable
-     */
-    // TODO Might want to add this one at some point. We would probably need a method to know the token types too.
-    // function tokensOfOwner(address _owner)
-    //     external
-    //     view
-    //     returns (uint256[] memory)
-    // {
-    //     uint256 tokenCount = balanceOf(_owner);
-    //     uint256[] memory tokensId = new uint256[](tokenCount);
-    //     for (uint256 i = 0; i < tokenCount; i++) {
-    //         tokensId[i] = tokenOfOwnerByIndex(_owner, i);
-    //     }
-
-    //     return tokensId;
-    // }
-
-    /**
-     * Withdrawing
+     * Withdraw balance from the contract
      */
-    // TODO Add the logic to distribute funds amongst the team
-    function withdraw() public payable onlyOwner {
+    function withdrawAll() public payable onlyOwner {
```

Comment:
that's where we should have the `reentrancy guard`?

Reply:
yes sir! added a TODO